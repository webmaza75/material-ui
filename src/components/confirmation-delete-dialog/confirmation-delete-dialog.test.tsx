import * as React from 'react';
import * as ShallowRenderer from 'react-test-renderer/shallow';

import ConfirmationDeleteDialog from './confirmation-delete-dialog';

describe(`ConfirmationDeleteDialog`, () => {
  it(`ConfirmationDeleteDialog renders correctly`, () => {
    const renderer = ShallowRenderer.createRenderer();
    const result = renderer
      .render(<ConfirmationDeleteDialog
        open={false}
        onTasksCancelDelete={jest.fn()}
        onTasksConfirmDelete={jest.fn()}
      />);

    expect(result).toMatchSnapshot();
  });
});
