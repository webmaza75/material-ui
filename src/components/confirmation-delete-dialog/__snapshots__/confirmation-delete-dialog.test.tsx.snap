// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfirmationDeleteDialog ConfirmationDeleteDialog renders correctly 1`] = `
<ForwardRef(WithStyles)
  aria-labelledby="alert-dialog-title"
  open={false}
>
  <ForwardRef(WithStyles)
    id="alert-dialog-title"
  >
    Delete selected tasks?
  </ForwardRef(WithStyles)>
  <ForwardRef(WithStyles)>
    <ForwardRef(WithStyles)
      color="primary"
      onClick={[MockFunction]}
    >
      Delete
    </ForwardRef(WithStyles)>
    <ForwardRef(WithStyles)
      autoFocus={true}
      color="primary"
      onClick={[MockFunction]}
    >
      Cancel
    </ForwardRef(WithStyles)>
  </ForwardRef(WithStyles)>
</ForwardRef(WithStyles)>
`;
